web/package.json

```diff
@@ -43,7 +43,7 @@
     "eslint-plugin-react-hooks": "4.3.0",
     "eslint-plugin-unused-imports": "^2.0.0",
     "http-server": "^14.0.0",
-    "i18next-parser": "^5.4.0",
+    "i18next-parser": "^3.3.0",
```

Comment:
Ho so this is the root cause of the `_plural` vs `_other` @mariomac following our [previous discussion](https://github.com/netobserv/network-observability-console-plugin/pull/38#discussion_r781936786) !